webextension/scripts/settings.js

```diff
@@ -144,11 +164,12 @@ function switchTabWindow() { $('input[type="radio"]').not(':checked').prop('chec
 
 function addDocs () {
```

Comment:
Merge incorrect: Revert to prior on most lines. Only add the 'show-resource-list' line. I'd like to reword it to: 'Display a list of resources during Save Page Now.'

Reply:
I have reverted the changes !!